braket_features/Getting_Devices_and_Checking_Device_Properties.ipynb

```diff
@@ -0,0 +1,990 @@
+      "      \"fXY_std_err\": 0.008696446383620746\n",
+      "    },\n",
+      "    \"31-32\": {\n",
+      "      \"fCZ\": 0.9660998724058678,\n",
+      "      \"fCZ_std_err\": 0.005761812906238927,\n",
+      "      \"fXY\": 0.9712881140159364,\n",
+      "      \"fXY_std_err\": 0.00806210364622841\n",
+      "    },\n",
+      "    \"32-33\": {\n",
+      "      \"fCZ\": 0.9829459414542475,\n",
+      "      \"fCZ_std_err\": 0.0022186828309007176,\n",
+      "      \"fXY\": 0.9670638653027691,\n",
+      "      \"fXY_std_err\": 0.0037801548720867483\n",
+      "    },\n",
+      "    \"33-34\": {\n",
+      "      \"fCZ\": 0.8849838977366044,\n",
+      "      \"fCZ_std_err\": 0.0052258548621047,\n",
+      "      \"fXY\": 0.9762641851192254,\n",
+      "      \"fXY_std_err\": 0.005696340189451561\n",
+      "    },\n",
+      "    \"34-35\": {\n",
+      "      \"fCZ\": 0.9044703600554278,\n",
+      "      \"fCZ_std_err\": 0.0058648284542988085,\n",
+      "      \"fXY\": 0.8369705573942525,\n",
+      "      \"fXY_std_err\": 0.0112620941149691\n",
+      "    },\n",
+      "    \"35-36\": {\n",
+      "      \"fCZ\": 0.945676821821138,\n",
+      "      \"fCZ_std_err\": 0.009523064811981776,\n",
+      "      \"fXY\": 0.8765771315623527,\n",
+      "      \"fXY_std_err\": 0.007783063940828758\n",
+      "    },\n",
+      "    \"36-37\": {\n",
+      "      \"fCZ\": 0.7935831809298916,\n",
+      "      \"fCZ_std_err\": 0.009437063268478572,\n",
+      "      \"fXY\": 0.9495459609794901,\n",
+      "      \"fXY_std_err\": 0.00763155830561965\n",
+      "    },\n",
+      "    \"4-5\": {\n",
+      "      \"fCZ\": 0.9044000502247598,\n",
+      "      \"fCZ_std_err\": 0.013536289599509059,\n",
+      "      \"fXY\": 0.9028915454624779,\n",
+      "      \"fXY_std_err\": 0.003875309777715083\n",
+      "    },\n",
+      "    \"5-6\": {\n",
+      "      \"fCZ\": 0.9748663918292196,\n",
+      "      \"fCZ_std_err\": 0.006644225111658855,\n",
+      "      \"fXY\": 0.9573385948832057,\n",
+      "      \"fXY_std_err\": 0.008834008846613742\n",
+      "    },\n",
+      "    \"6-7\": {\n",
+      "      \"fCZ\": 0.9597106644456523,\n",
+      "      \"fCZ_std_err\": 0.00915230083006339,\n",
+      "      \"fXY\": 0.959585777941374,\n",
+      "      \"fXY_std_err\": 0.006297039905522111\n",
+      "    }\n",
+      "  }\n",
+      "}\n"
+     ]
+    }
+   ],
+   "source": [
+    "# the IonQ device\n",
+    "device = AwsDevice(\"arn:aws:braket:::device/qpu/ionq/ionQdevice\")\n",
+    "\n",
+    "# the Rigetti device\n",
+    "device = AwsDevice(\"arn:aws:braket:::device/qpu/rigetti/Aspen-8\")\n",
+    "\n",
+    "execution_windows = device.properties.service.executionWindows\n",
+    "connectivity_graph = device.properties.paradigm.connectivity\n",
+    "calibration = device.properties.provider.specs\n",
+    "\n",
+    "print(f'The exeuation windows for {device.name}:\\n{execution_windows}\\n')\n",
```

Comment:
"print(f'The execution windows for {device.name}:\\n{execution_windows}\\n')\n",

(Note: I would prefer not to use the word execution, but in any case it has a typo. Could we substitute as follows?)

    "print(f'The availability windows for {device.name}:\\n{execution_windows}\\n')\n",

Reply:
yes

Reply:
excellent. Please do.